<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabcollegelist">
     <attribute name="title">
      <string>大学信息</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>学校名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leCollegeName">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入学校名</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>城市:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbProvince">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请选择省份</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCity">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请选择城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>学校类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbType">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请选择类型</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>学校性质</string>
      </property>
      <widget class="QCheckBox" name="cbTagsyl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>双一流</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbTag985">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>985</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbTag211">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>211</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbTagqjjh">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>强基计划</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="cbbxms">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>办学模式</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbzsms">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>招生模式</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvCollegeData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>821</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabgaokao">
     <attribute name="title">
      <string>大学招生详情</string>
     </attribute>
     <widget class="QLabel" name="lbIcon">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid blue;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbCollegeName">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>631</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>大学</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbDangci">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>278</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid blue;</string>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbAbstract">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>录取分数线</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbmajor">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>专业分数线</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvSchoolScore">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tvMajorScore">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>391</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSchoolDetail">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>学校详情</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMajor">
     <attribute name="title">
      <string>专业录取情况</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>录取年份：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbYear">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>专业名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtMajor">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>25</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入专业名，支持模糊查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMajor">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvMajor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>801</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPlan">
     <attribute name="title">
      <string>招生计划</string>
     </attribute>
     <widget class="QTableView" name="tvPlan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>801</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>学校名：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbPlanYear">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>25</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPlanSchool">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>25</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入学校名，支持模糊查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlan">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>录取年份：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>专业名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPlanMajor">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>25</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入专业名，支持模糊查询</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbProvince</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMajor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
